src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/cjsm/CjsmDirectoryRepositoryTest.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.cjsm
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.IntegrationTest
+
+class CjsmDirectoryRepositoryTest : IntegrationTest() {
+
+  @Test
+  fun `find by unique column secure email returns a nullable rather than a list`() {
+    cjsmDirectoryRepository.save(CjsmDirectoryEntry(secureEmail = "someone@company.com.cjsm.net", firstName = "Someone", lastName = "Someoneson", organisation = "Company", townCity = "Sometown", businessType = "Some business type"))
+
+    val directoryEntry = cjsmDirectoryRepository.findBySecureEmail("someone@company.com.cjsm.net")
+    assertThat(directoryEntry?.firstName).isEqualTo("Someone")
+
+    val noDirectoryEntry = cjsmDirectoryRepository.findBySecureEmail("neverheardofhim@company.com.cjsm.net")
```

Comment:
Minor, but is this 2 tests? 1 for happy path finding the record, 1 for sad path record not found?